core-bundle/src/Resources/contao/dca/tl_page.php

```diff
@@ -1150,6 +1150,10 @@ static function ($alias) use ($objPage, $aliasExists)
 				$varValue = $objPage->folderUrl . $varValue;
 			}
 		}
+		elseif (preg_match('/^[1-9]\d*$/', $varValue))
+		{
+			throw new Exception(sprintf($GLOBALS['TL_LANG']['ERR']['aliasNumeric'], $varValue));
```

Comment:
I think we need to add a translation for the `aliasNumeric` key.

Reply:
Added in b5243def42157a0c7aa1bffa8fd669a19af7f888. Is the check only needed for page aliases? What about news, events and FAQs?

Reply:
> What about news, events and FAQs?

Yes, we should problably add the check there too.

Reply:
Added in 1f9dc566781750b6e46aee7c8d2bbc694604cb8a.